lib/api/kuzzle.js

```diff
@@ -244,6 +246,19 @@ class Kuzzle extends EventEmitter {
       });
   }
 
+  adminExists () {
```

Comment:
I didn't know where to put this method, if somebody have a better idea I'm in

Reply:
Why not isolate it in its own file in the `lib/utils` directory?

Reply:
On this point, I agree with Benoit, I'm not comfortable on add another util. Moreover we will have to pass the kuzzle instance to this method, I prefer the OO way.
I think we miss some kind of `security` directory to group all classes/methods related to securities